english/God's-own-country/kerala-story.md

```diff
@@ -0,0 +1,4 @@
+Kerala, a state on India's tropical Malabar Coast, has nearly 600km of Arabian Sea shoreline.
+It's known for its palm-lined [beaches](../beaches/kerala2.md) and its [backwaters](../lagoons/kerala3.md), [rivers](../44rivers/kerala1.md) a network of canals popular for cruises. 
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.